src/index.ts

```diff
@@ -3,8 +3,6 @@ export * from "./lib/builder"
 export * from "./lib/apply-selector"
 export * from "./lib/types"
 export * from "./lib/utils"
-export * from "./lib/gerber"
-export * from "./lib/excellon-drill"
```

Comment:
can we add `export * from "circuit-json-to-gerber"` for backwards compatibility? It'll break less things

Reply:
ok